Core/include/Acts/Propagator/DirectNavigator.hpp

```diff
@@ -198,13 +179,12 @@ class DirectNavigator {
   template <typename propagator_state_t, typename stepper_t>
   void initialize(propagator_state_t& state,
                   const stepper_t& /*stepper*/) const {
-    ACTS_VERBOSE(volInfo(state) << "initialize");
```

Comment:
Imo having the volume info is pretty helpful in the log output.

EDIT: Ah, you're removing this because in *direct* we don't really have a current volume anyway?

Reply:
Right this will just print garbage in with the `DirectNavigator`